.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,7 @@
 - [ ] The description explains in detail what this PR is about.
 - [ ] I have linked a relevant issue or discussion.
 - [ ] I have created tests covering the changes.
-- [ ] I have updated the documentation accordingly.
+- [ ] I have updated the documentation and checked the generated documentation. <!-- You do not need to build the documentation locally. A bot is recreating the documentation whenever there is a change to this PR and will maintain a link to the latest preview once it's ready. -->
 
```

Comment:
The comment `<!-- ... -->` is helpful for a new sage developer but a visual noise to all others. Please remove the comment. 

Otherwise, I like the modified checklist.

Reply:
Or you may want "... and checked the documentation preview".

Reply:
> The comment `<!-- ... -->` is helpful for a new sage developer but a visual noise to all others. Please remove the comment.

I moved the remark to the generated comment. I think it's important to explain how these things work. I fear that if you are not contributing constantly you might not understand that these things actually update.

Reply:
Thanks.

Would you show a capture of the changed comment? 

Before I modified the comment to fit in a single line for easy read. I fear that the changed comment overflows to another line :-)

Reply:
Since the comment itself is not updated, it might not matter though.

Reply:
Yes, it does overflow. (Depends on your browser of course, but on my desktop setup I get this.)

![image](https://github.com/sagemath/sage/assets/373765/2d228079-18d2-4ec6-868c-2b86bc48a01d)

I don't see the problem with this overflowing to a second line though.

Reply:
I can change it to this if you prefer

![image](https://github.com/sagemath/sage/assets/373765/acdf0a66-3340-44cd-8a37-ac981682dcd1)

Reply:
OK with two lines.

Reply:
Is the commit number updated?

Reply:
> Is the commit number updated?

Sure.